.github/workflows/build.yml

```diff
@@ -4,7 +4,9 @@ jobs:
     name: build
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
@dbscoach @Adman I took the liberty of bumping this up to both make sure the `submodules` part works and that we don't get notified about it being deprecated.